packages/core-kit/src/language.ts

```diff
@@ -1,11 +1,6 @@
-import { z } from 'zod';
+import { LanguageTag, languageTagGuard } from '@logto/language-kit';
 
-import { fallback } from './utilities/zod';
+import { fallback } from '@/utilities';
 
-export const languageKeys = ['en', 'fr', 'pt-PT', 'zh-CN', 'tr-TR', 'ko-KR'] as const;
```

Comment:
Overall LGTM
Curious where can we get built-in languages?

Reply:
Do it like this:

```ts
export const isBuiltInLanguage = (key: string): key is LanguageKey =>
  Object.keys(resource).includes(key);
```